netmap/doc.go

```diff
@@ -1,11 +1,42 @@
 /*
-Package netmap provides routines for working with netmap and placement policy.
-Work is done in 4 steps:
-1. Create context containing results shared between steps.
-2. Processing filters.
-3. Processing selectors.
-4. Processing replicas.
-
-Each step depends only on previous ones.
+Package netmap provides functionality for working with information about the
+NeoFS network, primarily a layer of storage nodes.
+
+The package concentrates all the characteristics of NeoFS networks.
+
+NetMap represents NeoFS network map - one of the main technologies used to
+store data in the system. It is composed of information about all storage nodes
+(NodeInfo type) in a particular network. NetMap methods allow you to impose
```

Comment:
Here I just underline that there is no global NeoFS network, and all network maps are bound to `a particular network`. I didn't want to mention `epoch` here since actually `NetMap` doesn't care about epochs.